### ISSUE TITLE
Security upgrade file-type from 3.9.0 to 16.5.4

### ISSUE THREAD

**Issue by appendixshuffle** [postmanlabs/postman-collection#1285]
Hi
I would like to ask for reopening this [PR](https://github.com/postmanlabs/postman-collection/pull/1282) , which was closed without review. 
It is blocking our pipelines because of [CVE-2022-36313](https://www.cve.org/CVERecord?id=CVE-2022-36313)

**Comment by evanrolfe** [comment]
+1 we are also having this issue

**Comment by mtica** [comment]
+1 for this issue

**Comment by codenirvana** [comment]
I verified using [this test](https://github.com/sindresorhus/file-type/commit/d86835680f4cccbee1a60628783c36700ec9e254#diff-13876b4beb64b9f156474dc78f9c923952a7ca210d4507b6b3135bbe244f8a60R547) that `file-type@3.9.0` is not affected by the reported CVE, because the vulnerable code doesn't exist in this version.

I don't see this getting flagged using `npm audit` as well, which tool is reporting this version?

**Comment by alhalama** [comment]
This is being flagged by [snyk](https://security.snyk.io/vuln/SNYK-JS-FILETYPE-2958042) as a vulnerability.

**Comment by scotty6435** [comment]
Yeah, if the code is not present in this version then getting the CVE updated would be ideal. Then again, 3.9.0 is a liiitttle bit outdated so it's being flagged in our VMS on that basis alone. 3.9.0 is 7 years old and there's been 78 releases since that one.
